src/libraries/System.Net.NetworkInformation/tests/FunctionalTests/IPGlobalPropertiesTest.cs

```diff
@@ -138,6 +138,31 @@ public void IPGlobalProperties_TcpListeners_Succeed(IPAddress address)
             }
         }
 
+        [Theory]
+        [MemberData(nameof(Loopbacks))]
+        [SkipOnPlatform(TestPlatforms.Android, "Unsupported on Android")]
+        public void IPGlobalProperties_UdpListeners_Succeed(IPAddress address)
+        {
+            using (var server = new Socket(address.AddressFamily, SocketType.Dgram, ProtocolType.Udp))
+            {
+                server.Bind(new IPEndPoint(address, 0));
+                _log.WriteLine($"listening on {server.LocalEndPoint}");
+
+                IPEndPoint[] udpListeners = IPGlobalProperties.GetIPGlobalProperties().GetActiveUdpListeners();
+                bool found = false;
+                foreach (IPEndPoint ep in udpListeners)
+                {
+                    if (ep.Equals(server.LocalEndPoint))
+                    {
+                        found = true;
+                        break;
+                    }
+                }
+
+                Assert.True(found);
```

Comment:
```suggestion
                IPEndPoint[] udpListeners = IPGlobalProperties.GetIPGlobalProperties().GetActiveUdpListeners();
                Assert.Contains(listener, udpListeners);
```

Reply:
updated. This was copy from TCP so I updated that one as well.